internal/mode/static/state/graph/route_common.go

```diff
@@ -132,6 +164,33 @@ func CreateRouteKey(obj client.Object) RouteKey {
 	}
 }
 
+// CreateRouteKeyL4 takes a client.Object and creates a L4RouteKey.
+func CreateRouteKeyL4(obj client.Object) L4RouteKey {
```

Comment:
double check that this is used outside the package

Reply:
double checked and it is